gwvolman/utils.py

```diff
@@ -188,6 +188,8 @@ def _get_container_config(gc, tale):
         if tale['config']:
             tale_config.update(tale['config'])
 
+        digest=tale['imageInfo']['digest'] or ""
```

Comment:
Is this `or` clause intentional? or just leftover from debugging?

I don't think there is a case where we wouldn't want to populate the digest, right?